src/libs/logos.js

```diff
@@ -23,15 +23,15 @@ import { isAnvil, isBaseline, isBioDataCatalyst, isDatastage, isElwazi, isFirecl
 import * as Utils from 'src/libs/utils'
 
 
-export const getAppName = (longName = false) => Utils.cond(
+export const getAppName = ({ longName = false, capitalInitial = false } = { longName: false, capitalInitial: false }) => Utils.cond(
```

Comment:
Is this additional complexity _really_ necessary? Can we just always capitalize the T like with AnVIL, Baseline, and elwazi? It's surprising that this option is inconsistently applied.

Reply:
I was told by Product that RARE-X wanted their welcome text to be specifically `Welcome to the RARE-X Data Analysis Platform`, and not `Welcome to The RARE-X Data Analysis Platform` or `Welcome to RARE-X`.